### PR TITLE
Bump library versions and prepare v 1.0.1 of the Docker image

### PR DESCRIPTION
**Pull request by cyrille-leclerc**
Bump library versions and prepare v 1.0.1 of the Docker image

### CODE REVIEW

**Comment by cyrille-leclerc** [comment]
@reviewbybees

**Comment by schottsfired** [comment]
🐝  LGTM

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ We have decided to bundle many tools in the same image to cover as many Java use
 # Supported tags and respective `Dockerfile` links
 
 -   [`latest` (*latest/Dockerfile*)](https://github.com/cloudbees/java-build-tools-dockerfile/blob/master/Dockerfile)
+-   [`1.0.0` (*1.0.0/Dockerfile*)](https://github.com/cloudbees/java-build-tools-dockerfile/blob/1.0.1/Dockerfile)
```
Comment (ydubreuil) [inline README.md]: Should be `1.0.1` I guess

**Comment by ydubreuil** [comment]
Apart from the comment on the release notes, SGTM

**Comment by batmat** [comment]
LGTM :bee: 

I wonder if it wouldn't be even better with ENV all grouped at the top now you created those.

**Comment by cyrille-leclerc** [comment]
@ydubreuil thanks, I fixed the typo amending the commit

**Comment by cyrille-leclerc** [comment]
@batmat there is a benefit at keeping the ENV version next to the download piece as there are often cross checks and sometimes tweaks to the download URL

File: Dockerfile [hunk 2]
```diff
@@ -75,17 +75,23 @@ ENV MAVEN_HOME /usr/share/maven
...
+  && mv /usr/share/apache-ant-$ANT_VERSION /usr/share/ant \
   && ln -s /usr/share/ant/bin/ant /usr/bin/ant
 
 ENV ANT_HOME /usr/share/ant
 
 #==========
 # Selenium
 #==========
+
+ENV SELENIUM_MAJOR_VERSION 3.0
```
Comment (ndeloof) [inline Dockerfile]: could be defined as ${SELENIUM_VERSION%\.[1-9]}

Comment (cyrille-leclerc) [inline Dockerfile]: Thanks I'll try

File: Dockerfile [hunk 3]
```diff
@@ -151,26 +157,28 @@ RUN curl -sL https://deb.nodesource.com/setup_4.x | bash \
...
-RUN npm install --global azure-cli@0.10.1
+ENV AZURE_CLI_VERSION 0.10.8
+RUN npm install --global azure-cli@$AZURE_CLI_VERSION
 
 #====================================
 # BOWER, GRUNT, GULP
 #====================================
 
-RUN npm install --global grunt-cli@0.1.2 bower@1.7.9 gulp@3.9.1
+RUN npm install --global grunt-cli@1.2.0 bower@1.8.0 gulp@3.9.1
```
Comment (ndeloof) [inline Dockerfile]: no variable here to define versions ?

Comment (cyrille-leclerc) [inline Dockerfile]: Iterative process to improve the Dockerfile, I have not yet cleaned up these ones

**Pull request merged**